package-lock.json

```diff
@@ -20,10 +20,10 @@
       "dev": true,
       "requires": {
         "@babel/types": "7.0.0-beta.44",
-        "jsesc": "2.5.1",
```

Comment:
Do we need to update these?

Reply:
Probably something on my npm.  Anyway, I'm reverting this.